source/guides/azure/infiniband.md

```diff
@@ -257,10 +257,9 @@ Then start a new shell.
 
 Create a conda environment (see [UCX-Py](https://ucx-py.readthedocs.io/en/latest/install.html) docs)
 
-```shell
+````shell
 mamba create -n ucxpy {{ rapids_conda_channels }} {{ rapids_conda_packages }} ipython ucx-proc=*=gpu ucx ucx-py dask distributed numpy cupy pytest pynvml -y
 mamba activate ucxpy
-```
```

Comment:
This doesn't look right to me... were these removed by accident? Please check how this renders in the GitHub UI on your branch (https://github.com/rapidsai/deployment/blob/xgboost-azure/source/guides/azure/infiniband.md)... I can see there that removing this causes some text that's not intended to be code-formatting being represented in a code block.

<img width="1395" alt="Screenshot 2024-10-14 at 2 05 49 PM" src="https://github.com/user-attachments/assets/add549fe-abb0-4f08-ad6b-c56876847d31">

In case you're new to working in markdown, I've found these really helpful:

* https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
* https://yihui.org/en/2021/06/markdown-breath/

Reply:
thanks for the links!